camel/configs/nexa_config.py

```diff
@@ -0,0 +1,60 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from __future__ import annotations
+
+from typing import Sequence, Union
+
+from openai._types import NOT_GIVEN, NotGiven
+
+from camel.configs.base_config import BaseConfig
+
+
+class NexaConfig(BaseConfig):
+    r"""Defines the parameters for generating chat completions using Nexa API
+
+    Args:
+        temperature (float, optional): Sampling temperature to use, between
+            :obj:`0` and :obj:`2`. Higher values make the output more random,
+            while lower values make it more focused and deterministic.
+            (default: :obj:`0.7`)
+        max_tokens (int, optional): The maximum number of new tokens to
+            generate in the chat completion. The total length of input tokens
+            and generated tokens is limited by the model's context length.
+            (default: :obj:`1024`)
+        top_p (float, optional): An alternative to sampling with temperature,
+            called nucleus sampling, where the model considers the results of
+            the tokens with top_p probability mass. So :obj:`0.1` means only
+            the tokens comprising the top 10% probability mass are considered.
+            (default: :obj:`1.0`)
+        top_k (int, optional): The number of highest probability vocabulary
```

Comment:
If we are going to use openai compatibility, I think this parameter would not be accepted

Reply:
Since there is no evidence that nexa-sdk officially supports OpenAI serving (but it works well in my testing), do you think it is ok for us to delete `top_k` and `nctx` for now, and support them in the future if there are updates in nexa-sdk (and openai)?

Reply:
from https://github.com/NexaAI/nexa-sdk they mentioned that they offers an OpenAI-compatible API server